<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerInfoWidget</class>
 <widget class="QWidget" name="PlayerInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>209</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutAttribute">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="countriesLabel">
       <property name="text">
        <string>Countries: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="continentsLabel">
       <property name="text">
        <string>Continents: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="armiesLabel">
       <property name="text">
        <string>Armies: </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutValue">
     <property name="spacing">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="nameValueLabel">
       <property name="text">
        <string>CPU</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="countriesValueLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="continentsValueLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="armiesValueLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
